### PR TITLE
Fix for pi

### PR DESCRIPTION
**Pull request by cyberorg**

### CODE REVIEW

File: fixes/mss/20200609-msshome-brand [hunk 1]
```diff
@@ -12,5 +12,11 @@ fi
 glib-compile-schemas /usr/share/glib-2.0/schemas/
 update-initramfs -u
 if [ $(arch) != x86_64 ]; then
-    echo "todo pi splash part" 
+    if ! grep -q plymouth; then
+        sed -i -e 's@$@ quiet splash plymouth.ignore-serial-consoles@' /boot/firmware/cmdline.txt
+    fi
+    cp /boot/firmware/initrd.img /boot/firmware/initrd.img.$(date +%m%d%H%M)
```
Comment (intelliant01) [inline fixes/mss/20200609-msshome-brand:18]: Backup and copying over to /boot/firmware is being taken care of by `update-initramfs -u`. So the following 4 lines are redundant.

**Comment by intelliant01** [comment]
Have also confirmed that the splash changes shall persist across kernel upgrades because update-initramfs shall pickup the alternative default plymouth set.

**Comment by cyberorg** [comment]
Plymouth default had already been set, and update-initramfs -u run, so why was old branding shown after raspi changes till cp from /boot to /boot/firmware?

**Comment by intelliant01** [comment]
The update-initramfs's function to copy over may have been fixed in a package update applied after we tried the script last time. Which package contains update-initramfs?

**Pull request merged**